fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/responses/ViewHelpers.java

```diff
@@ -122,7 +123,7 @@ public static ViewHelpers getInstance() {
         final Node subject, final Resource predicate) {
         final Iterator<Triple> versions = getObjects(graph, subject, predicate);
         final Map<String, Node> map = new TreeMap<>();
-        final SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
+        final SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'", Locale.US);
```

Comment:
`Locale.US` will probably not make sense for the rest of the world that uses Fedora. In this case, allowing `SimpleDateFormat` to use its default `Locale` is probably fine.